src/queries/role.ts

```diff
@@ -134,11 +150,14 @@ function queryAllByRole(
       return true
     })
     .filter(element => {
-      return hidden === false
-        ? isInaccessible(element, {
+      return hidden
+        ? true
+        : !isInaccessible(element, {
```

Comment:
If the first branch of your ternary is `true` you can just use `||` to provide the fallback
i.e. `A ? true : B` is equivalent to `A || B`.
```suggestion
      return hidden || !isInaccessible(element, {
```